doc/whats_new.rst

```diff
@@ -255,6 +255,12 @@ API changes summary
       selection classes to be used with tools such as
       :func:`sklearn.model_selection.cross_val_predict`.
       :issue:`2879` by :user:`Stephen Hoover <stephen-hoover>`.
+      
+   - Deprecate the ``Y`` parameter from ``transform`` method signature in 
```

Comment:
Can you format this correctly. Respect the indentation whatever the number of spaces is. I don't think you need to list all the classes.

Also this is a small `y` not a capital one